gcf_data_mapper/parsers/helpers.py

```diff
@@ -26,3 +26,64 @@ def verify_required_fields_present(
 def check_required_column_value_not_na(row: pd.Series, column_enum) -> bool:
     """Check if the row contains valid document column values (not NA)."""
     return all(pd.notna(row[column.value]) for column in column_enum)
+
+
+def row_contains_columns_with_empty_values(
+    row: pd.Series, required_columns: list[str]
+) -> bool:
+    """Check that all required values in the given row are not empty (isna).
+
+    :param pd.Series row: The row to check for isna values.
+    :param list[str] required_columns: A list of column names that will be used to verify
+        isna values.
+    :return bool: True if the row contains columns with empty values, false if if all
+        expected columns are populated
+    """
+
+    # Ensure we are working with a pandas Series by re-selecting the required columns as a Series
+    row_subset = pd.Series(row[required_columns], index=required_columns)
+
+    if row_subset.isna().any():
```

Comment:
I LOVE this function - so much better than the way I'd done it in the events parser!